dotcom-rendering/src/lib/decideCardContentPadding.test.ts

```diff
@@ -0,0 +1,61 @@
+import { space } from '@guardian/source/foundations';
+import type { ImagePositionType } from '../components/Card/components/ImageWrapper';
+import { decideCardContentPadding } from './decideCardContentPadding';
+
+describe('decideCardContentPadding', () => {
+	const testCases = [
+		{
+			args: {
+				imagePositionOnMobile: 'none' as ImagePositionType,
+				imagePositionOnDesktop: 'top' as ImagePositionType,
+				addAdditionalPadding: false,
+			},
+			expectedResult: {
+				mobilePadding: '0px',
+				desktopPadding: `${space[2]}px 0px 0px 0px`,
+			},
+		},
+		{
+			args: {
+				imagePositionOnMobile: 'bottom' as ImagePositionType,
+				imagePositionOnDesktop: 'left' as ImagePositionType,
+				addAdditionalPadding: false,
+			},
+			expectedResult: {
+				mobilePadding: `0px 0px ${space[2]}px 0px`,
+				desktopPadding: `0px 0px 0px ${space[2]}px`,
+			},
+		},
+		{
+			args: {
+				imagePositionOnMobile: 'right' as ImagePositionType,
+				imagePositionOnDesktop: 'left' as ImagePositionType,
```

Comment:
FYI: you could have gone over this with either of the following operators on `testCases`: `as const` or `satisfies Array<{ args: Parameters<typeof decideCardContentPadding>[0], expextedResult: Record<string, string>}>`